server/data/data-api/src/main/java/org/apache/james/rrt/api/CanSendFrom.java

```diff
@@ -30,6 +33,10 @@ public interface CanSendFrom {
      */
     boolean userCanSendFrom(Username connectedUser, Username fromUser);
 
+    default Publisher<Boolean> userCanSendFromReactive(Username connectedUser, Username fromUser) {
```

Comment:
CanSendFromImpl don't need to wrap it's blocking calls? Shouldn't it implement this default method?